script/test/fixtures/etherpad/docker-compose.yml

```diff
@@ -6,10 +6,10 @@ services:
     ports:
       - "$DB_PORT"
     environment:
-      MYSQL_ROOT_PASSWORD: $ROOT_PASS
-      MYSQL_DATABASE: $DB_NAME
-      MYSQL_PASSWORD: $DB_PASS
-      MYSQL_USER: $DB_USER
+      - "MYSQL_ROOT_PASSWORD=$ROOT_PASS"
```

Comment:
You're still changing these values!

Leave them be please, no quotes, no editing them. They should stay the same as before.